### ISSUE TITLE
Object reference not set to an instance of an object.

### ISSUE THREAD

**Issue by Rarisma** [storybuilder-org/StoryCAD#629]
Time: 09/12/2023 23:17:05
Application: 
Version: Version: 2.12.0.0
Severity: Fatal
URL:  :arrow_right:  :arrow_right: 
Error: `System.NullReferenceException`
Source: StoryCADLib
User: seanf
Hostname: ST_SHADOW

More details on [elmah.io](https://app.elmah.io/errorlog/search?logId=6e87ad13-67cd-460b-b72f-0866cc738d89&hidden=true&expand=true&filters=id:%22513EDB70A08203FC%22#searchTab)

**Comment by terrycox** [comment]
Sean's comments:
Additionally, navigating to the Development tab of more than one scene within a story crashes StoryCAD, but only in a new/user-created story, not in the (tested) sample stories.  "Object reference not set to an instance of an object".  The crash does not occur when the Development tab is active while the user switches between scene elements - in that case, the Development data populates as expected.  That's probably hard to follow - the non-crashing switch between scenes is in the video at 50 seconds in. (edited)

The video he mentions is attached:
[2023-09-12_20-03-20.zip](https://github.com/storybuilder-org/StoryCAD/files/12595216/2023-09-12_20-03-20.zip)

**Comment by Rarisma** [comment]
fixed in #632
